examples/DotNetCore/WebDemoWithEventHub/WebDemoWithEventHub/Startup.cs

```diff
@@ -20,8 +21,15 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddRazorPages();
 
-            // We add a Settings model to the service container, which takes its values from the applications configuration.
+            // We add a Settings and EventHubConnection models to the service container, which takes its values from the applications configuration.
             services.Configure<Settings>(Configuration.GetSection("WebDemo:Settings"));
+            services.Configure<EventHubConnection>(Configuration.GetSection("WebDemo:EventHubConnection"));
+
+            // Adds Azure App Configuration services to the specified Microsoft.Extensions.DependencyInjection.IServiceCollection.
```

Comment:
Let's follow the existing style. How about?

"Add Azure App Configuration required services to the DI".

Reply:
Updated the comment.